src/components/Button/Button.module.css

```diff
@@ -0,0 +1,35 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 4px;
+}
+
+.buttonSm {
+  display: flex;
+  align-items: center;
+  padding: 4px 12px !important;
+}
+
+.buttonMd {
+  display: flex;
+  align-items: center;
+  padding: 8px 12px !important;
+}
+
+.buttonLg {
+  display: flex;
+  align-items: center;
+  height: 48px !important;
+  padding: 16px !important;
+}
+
+.buttonGhost {
+  border-color: transparent !important;
```

Comment:
Can we use the token defaultGhostBorderColor?

Reply:
Unfortunately not, because that token affects the behavior of `ghost` only for what Ant defines as the default button (what would be our neutral). So for ghost primary and danger, the style would not be applied

Reply:
I see. ok for now then :+1: